sig-testing/flakiness-sla.md

```diff
@@ -0,0 +1,40 @@
+# Kubernetes Testing Flakiness SLA
+
+This document captures the expectations of the community about flakiness in
+our tests and our test infrastructure. It sets out an SLA (Service Level 
+Agreement) for flakiness in our tests, as well as actions that we will
+take when we are out of SLA.
+
+
+## Definition of a "Flake"
+
+We'll start by the definition of a _flake_. A test result is considered to
```

Comment:
>test result

what's the right level of granularity here?

- job - eg: pull-kubernetes-e2e-gke-gci
- suite - eg: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/ingress.go
- test case - eg: https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/42391/pull-kubernetes-e2e-gke-gci/20582/#k8sio-loadbalancing-l7-k8sio-nginx-should-conform-to-ingress-spec